project.clj

```diff
@@ -12,6 +12,7 @@
   ;; dependencies. Also supports :warn to simply emit warnings.
   ;; requires lein 2.2.0+.
   :pedantic? :abort
+  :packaging "pom"
```

Comment:
what's this?

Reply:
parent poms are supposed to have packaging `pom` and not `jar` to follow Maven style specs. If you were to try and use clj-parent from a Maven project as a parent pom, it would fail because of this.